.eslintrc

```diff
@@ -82,16 +79,21 @@
       {
         "ignoreRestSiblings": true
       }
-    ]
+    ],
+    // disabled rules for airbnb config v18
+    "react/state-in-constructor": 0,
+    "react/jsx-curly-newline": 0,
+    "no-multiple-empty-lines": 0,
+    "prefer-object-spread": 0
   },
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
   "parserOptions": {
     "sourceType": "module",
-    "ecmaVersion": 8,
+    "ecmaVersion": 2020,
     "ecmaFeatures": {
-      "jsx": true,
-      "experimentalObjectRestSpread": true
```

Comment:
removed in eslint v6.
ref: https://eslint.org/docs/latest/use/migrating-to-6.0.0#-the-deprecated-experimentalobjectrestspread-option-has-been-removed